src/smt/smt_engine.cpp

```diff
@@ -680,8 +668,7 @@ void SmtEngine::defineFunctionsRec(
       lem = nm->mkNode(kind::FORALL, boundVars, lem, aexpr);
     }
     // assert the quantified formula
-    //   notice we don't call assertFormula directly, since this would
-    //   duplicate the output on raw-benchmark.
+    //   notice we don't call assertFormula directly.
```

Comment:
Is there another reason to not call `assertFormula` directly now?

Reply:
Haha I thought you'd ask, the general reason is that we shouldn't use the SmtEngine interface from itself.